src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1228,7 +1228,8 @@ struct RPCCallImp
 
             // Parse reply
             JLOG (j.debug()) << "RPC reply: " << strData << std::endl;
-
+            if (strData.find("Unable to parse request") == 0)
+                Throw<std::runtime_error> (strData);
```

Comment:
I don't know if's any better, but could we throw a `std::invalid_argument` ex and catch that specific type below instead of checking the message text?

Reply:
Thanks for the comments and the code!  I just wanted you guys to know that I'm not ignoring this, and will accept Scott's generous update.  I'm just concentrating on another PR at the moment.